themes/jamstackthemes/layouts/partials/stackbit.html

```diff
@@ -4,4 +4,4 @@
   <img src="{{ "images/stackbit.png" | relURL }}" alt="Stackbit Logo" />
 </div>
 
-<span class="made-by made-by-mobile"><a href="https://www.stackbit.com"><img src="{{ "images/stackbit.png" | relURL }}" alt="Stackbit Logo" /></a></span>
\ No newline at end of file
+<span class="made-by made-by-mobile"><a href="https://www.stackbit.com/&utm_source=jamstackthemes&utm_medium=referral&utm_campaign=footer"><img src="{{ "images/stackbit.png" | relURL }}" alt="Stackbit Logo" /></a></span>
```

Comment:
same as above. `utm_campaign` should be `jamstackthemes` and `utm_content` should be `footer`